scalastyle-config.xml

```diff
@@ -101,5 +90,38 @@
     </check>
     <check level="error" class="org.scalastyle.scalariform.PublicMethodsHaveTypeChecker" enabled="true"></check>
     <check level="error" class="org.scalastyle.file.NewLineAtEofChecker" enabled="true"></check>
+
+    <!--=============================================================================================================-->
+    <!--                                              temporary rules                                                -->
+    <!--=============================================================================================================-->
+    <check level="warn" class="org.scalastyle.scalariform.IllegalImportsChecker" enabled="true">
+        <parameters>
+            <parameter name="illegalImports"><![CDATA[org.joda._]]></parameter>
+        </parameters>
+    </check>
+
+    <!--=============================================================================================================-->
+    <!--                                     rules we don't want to enforce yet                                      -->
+    <!--=============================================================================================================-->
+    <check enabled="false" class="org.scalastyle.scalariform.UnderscoreImportChecker" level="warning">
```

Comment:
nit: other rules have the `level` property first and `enabled` at the end - consider keeping properties in the same order throughout.

Reply:
Do we need an XML style checker? 😁

Reply:
Yes, but we'll use json for defining rules.